src/parser/ad_parser.js

```diff
@@ -116,7 +117,9 @@ function parseAdElement(adTypeElement, emit) {
           from extensions the same way than for an AdVerifications node.
         */
         if (!ad.adVerifications.length) {
-          ad.adVerifications = _parseAdVerificationsFromExensions(extNodes);
+          adVerificationFromExtensions = _parseAdVerificationsFromExensions(
```

Comment:
Can you fix this typo ? 
```suggestion
          adVerificationFromExtensions = _parseAdVerificationsFromExtensions(
```